Tests/ApolloServerIntegrationTests/URLSessionClientTests.swift

```diff
@@ -4,7 +4,19 @@ import ApolloUtils
 
 class URLSessionClientLiveTests: XCTestCase {
   
-  let client = URLSessionClient()
+  var client: URLSessionClient!
+
+  override func setUp() {
+    super.setUp()
+
+    client = URLSessionClient()
+  }
+
+  override func tearDown() {
+    client = nil
+
+    super.tearDown()
+  }
```

Comment:
is this defensive or is there an actual problem here? I ask because I've never seen this be an issue when a `let` of `URLSessionClient`, and this would create a new instance per test run rather than reusing an existing variable.

Reply:
Nope it's actually leaking! I'll call and explain why! (Each test is already creating a new instance currently!)

Reply:
If you wanted to reuse the same client you would need this to be `static let`, just `let` won't do it because each test initializes it's own instance of the `XCTestCase` subclass.